ext/opentelemetry-ext-dbapi/src/opentelemetry/ext/dbapi/__init__.py

```diff
@@ -184,7 +193,7 @@ def get_connection_attributes(self, connection):
             self.name += "." + self.database
         user = self.connection_props.get("user")
         if user is not None:
-            self.span_attributes["db.user"] = user
+            self.span_attributes["db.user"] = str(user)
```

Comment:
User is not returned as string in some cases?

Reply:
Right, PyMySQL returns bytes and it was causing problems in the console exporter. I opened an issue about it https://github.com/open-telemetry/opentelemetry-python/issues/623.